examples/chef/chef.py

```diff
@@ -489,6 +489,10 @@ def main(argv: Sequence[str]) -> None:
     #
 
     if options.do_build:
+        branch = shell.run_cmd(
+            "git branch | awk -v FS=' ' '/\*/{print $NF}' | sed 's|[()]||g'", return_cmd_output=True).replace("\n", "")
+        commit_id = shell.run_cmd("git rev-parse HEAD", return_cmd_output=True).replace("\n", "")
```

Comment:
Thanks for suggestion, I'll consider that in the future!

Reply:
This is too confusing, now changing branches will affect the resulting binary.

This type of stamping should be opt-in, not opt-out, since it's also seriously detrimental to performance and artifact caching since even a documentation change will result in a different binary.